x-pack/plugins/uptime/public/components/common/charts/__snapshots__/donut_chart.test.tsx.snap

```diff
@@ -80,6 +80,24 @@ exports[`DonutChart component passes correct props without errors for valid prop
                 "strokeWidth": 1,
                 "visible": false,
               },
+              "lumaSteps": Array [
```

Comment:
Will this affect the rendered donut chart visually? I may need to pull down this branch to do a feature-level review, as I'm not sure if this will cause a visual regression.

Reply:
no the `lumaSteps` are just used in XY charts for the multilayer time axis. This array is part of the `Theme` object so it's actually just reflected in the jest snapshot as it is.